siliconcompiler/core.py

```diff
@@ -1123,7 +1132,13 @@ def _find_files(self,
         if search_paths:
             search_paths = self.__convert_paths_to_posix(search_paths)
 
-        for path in paths:
+        while len(dependencies) < len(paths):
+            dependencies.append(None)
+
+        for (dependency, path) in zip(dependencies, paths):
+            if dependency:
+                result.append(os.path.join(dep.path(self, dependency), path))
+                continue
```

Comment:
we need some error messaging here is the file is not found in the declared package

Reply:
Added